### ISSUE TITLE
Compilation error with rust 1.40

### ISSUE THREAD

**Issue by ln42** [RustAudio/rust-lv2#81]
Hey there, thanks for the awesome work on lv2 support with Rust!

I just got back on a project I left a couple weeks and it's not compiling anymore:
```
error[E0432]: unresolved import `proc_macro`
 --> /.cargo/registry/src/github.com-1ecc6299db9ec823/urid-derive-0.1.0/src/lib.rs:7:5
  |
7 | use proc_macro::TokenStream;
  |     ^^^^^^^^^^ help: a similar path exists: `syn::proc_macro`

error[E0432]: unresolved import `proc_macro`
 --> /.cargo/registry/src/github.com-1ecc6299db9ec823/urid-derive-0.1.0/src/uri_bound.rs:2:5
  |
2 | use proc_macro::TokenStream;
  |     ^^^^^^^^^^ help: a similar path exists: `syn::proc_macro`

error[E0432]: unresolved import `proc_macro`
 --> /.cargo/registry/src/github.com-1ecc6299db9ec823/urid-derive-0.1.0/src/urid_collection_derive.rs:1:5
  |
1 | use proc_macro::TokenStream;
  |     ^^^^^^^^^^ help: a similar path exists: `syn::proc_macro`

error: aborting due to 3 previous errors
```

I'll try to fix it myself by cloning the repo locally, but I thought it was worth reporting as I tried with latest `develop` and had the same error.

Cheers

**Comment by ln42** [comment]
Fixed with https://github.com/RustAudio/rust-lv2/pull/83

**Comment by Janonard** [comment]
I can reproduce this error. This problem was fixed [back in March with 1.42.0](https://blog.rust-lang.org/2020/03/12/Rust-1.42.html#use-proc_macrotokenstream-now-works), but I will merge the fix nonetheless. However, I would kindly ask you to update your toolchain! :sweat_smile: We haven't made it official, but in practice we are only targeting the currently stable versions of Rust, which is 1.46.0 as of this writing.
